mnefun/_mnefun.py

```diff
@@ -239,6 +240,11 @@ class Params(Frozen):
     sss_origin : array-like, shape (3,) | str
         Origin of internal and external multipolar moment space in meters.
         Default is center of sphere fit to digitized head points.
+    weighting : bool | False
```

Comment:
`bool`

But this isn't a great future-compatible mode. I'd probably make it `str | None` and allow only `'chpi'` as a string option for now. And maybe it should be called `average_weighting` since it's only used for creating evoked